packages/devtools_app/lib/src/screens/performance/performance_screen.dart

```diff
@@ -369,6 +369,30 @@ class MoreDebuggingOptionsButton extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    final theme = Theme.of(context);
+    final textStyle = theme.subtleTextStyle;
+    final overlayChildren = <Widget>[
+      Text(
+        'When toggling on/off a rendering layer, you will need '
+        'to reproduce activity in your app to see the effects of the '
+        'debugging option. All layers are rendered by default - disabling a '
+        'layer may help you identify expensive operations in your app.',
+        style: Theme.of(context).subtleTextStyle,
+      )
+    ];
+    if (!serviceManager.connectedApp!.isDebugFlutterAppNow) {
+      overlayChildren.addAll([
+        const SizedBox(height: denseSpacing),
+        RichText(
+          text: TextSpan(
+            text:
+                'These debugging options are not available for a profile build. To use them, run your app in debug mode.',
```

Comment:
Maybe consider rewording "not available for a profile build" to "not available in profile mode"?

Reply:
I would think that these should automatically be hidden or disabled if the service extension isn't registered. Is that logic going wrong (e.g. Flutter is registering the service extensions but making them no-ops?)?

Reply:
The checkboxes are already disabled. It's not obvious because we already use a subtle text here. There will be a follow up issue to make the checkboxes checked by default, as that is the default state. Having the checkboxes checked with a gray disabled color will make it more obvious that they are already disabled